README.md

```diff
@@ -49,7 +49,7 @@ app.post('/graphql', graphqlHTTP({
 
 app.get('/graphql', graphqlHTTP({
   schema: MyGraphQLSchema,
-  graphiql: true // set it to false to disable the GraphiQL interface
```

Comment:
the reason for this comment was to make it clear that setting `graphiql` to true is not mandatory just because this is a GET method.
Setting this to false will still allow to serve GraphQL requests through GET with query params.